_posts/2021-05-21-libera.markdown

```diff
@@ -0,0 +1,15 @@
+---
+title: IRC network switch to Libera
+layout: post
+---
+
+Given the recent [upheaval at Freenode](https://lwn.net/Articles/856543/), the
+Void Linux team has made the decision to switch to another IRC network. Our
+network of choice is <https://libera.chat>, managed by former Freenode staff.
+When accessing `#voidlinux` and `#xbps`, be sure to use `irc.libera.chat`! We
```

Comment:
maybe #voidlinux and #xbps could be ircs:// links

Reply:
I don't think it's necessary, but if others think the same I can change.